client/src/js/app.js

```diff
@@ -206,22 +198,20 @@ function translateConfig($translateProvider) {
 }
 
 function localeConfig(tmhDynamicLocaleProvider) {
-
   // TODO Hardcoded default translation/ localisation
   tmhDynamicLocaleProvider.localeLocationPattern('/i18n/locale/angular-locale_{{locale}}.js');
   tmhDynamicLocaleProvider.defaultLocale('fr-be');
 }
 
 // redirect to login if not signed in.
 function startupConfig($rootScope, $state, $uibModalStack, SessionService, amMoment, Notify, $location) {
-
   var loginStateRegexp = /#\/login$/;
   var rootStateRegexp = /#\/$|\/$|#$/;
 
 
   // make sure the user is logged in and allowed to access states when
   // navigating by URL.  This is pure an authentication issue.
-  $rootScope.$on('$locationChangeStart', function (event, next) {
+  $rootScope.$on('$locationChangeStart', (event, next) => {
```

Comment:
Unfortunately, we cannot use arrow functions in the browser.  They break old versions of chrome and break our gulp build :-1: